modules/ztp-image-based-upgrade-installing-lifecycle-agent-using-cli.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-lcao-using-cli_{context}"]
+= Installing the {lcao} by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the {lcao} on both the seed and target cluster.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the {rh-rhacm} Operator 2.9.2 or later.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `Subscription` CR:
+.. Define the `Subscription` CR and save the YAML file, for example, `lcao-subscription.yaml`:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-lifecycle-agent-subscription
+  namespace: openshift-operators
+spec:
+  channel: alpha
+  name: lifecycle-agent
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+.. Create the `Subscription` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f lcao-subscription.yaml
+----
+
+.Verification
+
+. Verify that the installation succeeded by inspecting the CSV resource:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-operators
```

Comment:
```suggestion
$ oc get csv -n openshift-lifecycle-agent
```

@amolnar-rh should be the `openshift-lifecycle-agent` namespace. This is a namespaced operator, different than TALM for instance which is installed in all namespaces.

Reply:
see suggestion in https://github.com/openshift/openshift-docs/pull/69809#discussion_r1504698301